apps/explorer/src/app/routes/assets/index.tsx

```diff
@@ -1,51 +1,108 @@
-import { getNodes, t } from '@vegaprotocol/react-helpers';
-import React from 'react';
+import { t } from '@vegaprotocol/react-helpers';
 import { RouteTitle } from '../../components/route-title';
-import { SubHeading } from '../../components/sub-heading';
-import { Loader, SyntaxHighlighter } from '@vegaprotocol/ui-toolkit';
-import { useExplorerAssetsQuery } from './__generated__/Assets';
-import type { AssetsFieldsFragment } from './__generated__/Assets';
+import type { VegaICellRendererParams } from '@vegaprotocol/ui-toolkit';
+import { AsyncRenderer } from '@vegaprotocol/ui-toolkit';
+import { ButtonLink } from '@vegaprotocol/ui-toolkit';
 import { useScrollToLocation } from '../../hooks/scroll-to-location';
 import { useDocumentTitle } from '../../hooks/use-document-title';
-import EmptyList from '../../components/empty-list/empty-list';
+import type { AssetFieldsFragment } from '@vegaprotocol/assets';
+import {
+  AssetStatusMapping,
+  AssetTypeMapping,
+  useAssetDetailsDialogStore,
+} from '@vegaprotocol/assets';
+import { useAssetsDataProvider } from '@vegaprotocol/assets';
+
+import { AgGridDynamic as AgGrid } from '@vegaprotocol/ui-toolkit';
+import type { AgGridReact } from 'ag-grid-react';
+import { AgGridColumn } from 'ag-grid-react';
+import { useRef } from 'react';
 
 const Assets = () => {
-  const { data, loading } = useExplorerAssetsQuery();
   useDocumentTitle(['Assets']);
-
   useScrollToLocation();
+  const { open: openAssetDetailsDialog } = useAssetDetailsDialogStore();
 
-  const assets = getNodes<AssetsFieldsFragment>(data?.assetsConnection);
-
-  if (!assets || assets.length === 0) {
-    if (!loading) {
-      return (
-        <section>
-          <RouteTitle data-testid="assets-header">{t('Assets')}</RouteTitle>
-          <EmptyList
-            heading={t('This chain has no assets')}
-            label={t('0 assets')}
-          />
-        </section>
-      );
-    } else {
-      return <Loader />;
-    }
-  }
+  const { data, loading, error } = useAssetsDataProvider();
+  const ref = useRef<AgGridReact>(null);
+  const table = (
```

Comment:
**nitpick (non-blocking)**:  This Asset list inherits a bug that I've also seen on Governance that @sam-keen recently put in a fix for. If you size the window down to mobile-ish size, there's a large gap between the title (`Assets`) and the header. 

Mobile view is not a priority currently, but if there's a fix for this that's easy to slot it, let's slot it in before merging

Reply:
It looks like it's `nav` related. It should be fixes once we change our sidebar to a proper menu :-)